docs/publications/nionswift2020/abstract.rst

```diff
@@ -0,0 +1,44 @@
+LiberTEM
+========
+
+**LiberTEM creators**: Alexander Clausen¹, Dieter Weber¹, Karina Ruzaeva¹, Vadim
+Migunov¹², Jan Caron¹, Rahul Chandra³, Magnus Nord⁴, Colin Ophus⁵, Simon Peter,
+Jay van Schyndel⁶, Jaeweon Shin⁷, Knut Müller-Caspary¹, Rafal Dunin-Borkowski¹
+
+**Presentation**: Dieter Weber¹
+
+¹Jülich Research Centre, ²RWTH Aachen University, ³Chandigarh University,
+⁴University of Antwerp, ⁵Lawrence Livermore National Lab, ⁶Monash University
+eResearch Centre, ⁷ETH Zürich
+
+Starting from 2009, the data rate of TEM cameras has outpaced the development of
+network, mass storage and memory bandwidth by two orders of magnitude. This
+immense increase in performance opens new doors for applications and, at the
+same time, requires adequate IT systems to handle acquisition, storage and
+processing that go well beyond solutions based on personal computers.
+
+The LiberTEM open source platform is developed to address these challenges. It
+allows high-throughput distributed processing of large-scale binary data sets
+using a simplified MapReduce programming model. This programming model can
+generate a stream of intermediate results for interactive GUI display, allows to
+process live data streams and offline data with the same algorithms, and scales
```

Comment:
I think this sentence is missing a word: "This programming model can generate a stream of intermediate results for interactive GUI display, **allows to process** live data streams and offline data with the same algorithms..."

Specifically the `allows to process` part.

Reply:
Thx @magnunor! I've changed the sentence. Is this better?